src/chart.zoom.js

```diff
@@ -102,23 +108,36 @@ function zoomTimeScale(scale, zoom, center) {
 	var minDelta = newDiff * min_percent;
 	var maxDelta = newDiff * max_percent;
 
-	options.time.min = scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) + minDelta);
-	options.time.max = scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - maxDelta);
+	var newMin = scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) + minDelta);
+	options.time.min = newMin < min_limit ? min_limit : newMin;
```

Comment:
Anything that parses as a valid label could be used so it could be a moment object, or it could be a string, or it could be a date object. Your best bet would be to try and use moment to abstract that away and then use the methods supplied by moment like `diff` to figure out which is the minimum value